acft/environments/acft-ubuntu20.04-py38-pytorch1.13.1-cuda11.7-gpu/asset.yaml

```diff
@@ -0,0 +1,10 @@
+name: acft-ubuntu20.04-py38-pytorch1.13.1-cuda11.7-gpu
```

Comment:
What's the purpose of this environment? The name doesn't adhere to [naming conventions](https://github.com/Azure/azureml-assets/wiki/Asset-naming-convention#environments), but if it's largely being used to run finetuning components we can deviate a bit.